app/bot/handlers/private/enable.py

```diff
@@ -0,0 +1,46 @@
+import logging
+from typing import Optional
+
+from aiogram.types import Message
+
+from app.enums.telegram_source import TelegramSource
+from app.services.exceptions.response_exception import ResponseException
+from app.services.telegram_group_api import TelegramGroupAPI
+from app.services.types.telegram_groups import (
+    TelegramGroups,
+)
+from app.services.types.teleram_group import (
+    CreateTelegramGroup,
+    TelegramGroup,
+    UpdateTelegramGroup,
+)
+from app.services.user_api import UserAPI
+
+
+async def enable(message: Message) -> None:
+    async with UserAPI() as user_api:
+        user = await user_api.get_user_by_telegram_id(message.from_user.id)  # type: ignore[union-attr]
+    try:
+        async with TelegramGroupAPI() as telegram_group_api:
+            telegram_groups: TelegramGroups = await telegram_group_api.get_telegram_groups(user.group.id)
+            telegram_group: Optional[TelegramGroup] = next(filter(lambda x: all((x.source == TelegramSource.PERSONAL_CHAT, x.telegram_id == message.from_user.id)), telegram_groups.telegram_groups), None) # type: ignore
```

Comment:
`all((x.source == TelegramSource.PERSONAL_CHAT, x.telegram_id == message.from_user.id))` replace to `(x.source == TelegramSource.PERSONAL_CHAT and x.telegram_id == message.from_user.id)`